src/pages/CreatePool/CreatePoolDialog.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  Alert,
+  Box,
+  Button,
+  Dialog,
+  DialogContent,
+  Divider,
+  Stack,
+  Typography,
+} from "@mui/material"
+import DialogTitle from "../../components/DialogTitle"
+import React from "react"
+import { useTranslation } from "react-i18next"
+
+type Props = { open: boolean; onClose?: () => void }
+
+export default function ReviewCreatePool({
+  open = true,
+  onClose,
+}: Props): JSX.Element {
+  const { t } = useTranslation()
+
+  return (
+    <Dialog open={open}>
+      <DialogTitle variant="h1" onClose={onClose}>
```

Comment:
What does `onClose` on the `DialogTitle` do?

Reply:
It close modal.

Reply:
Clicking on the DialogTitle closes it? onClose handler should be on `Dialog`.

Reply:
OK, let me do it on another branch.